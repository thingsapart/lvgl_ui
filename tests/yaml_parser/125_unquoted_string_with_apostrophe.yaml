# Tests that an apostrophe can exist in an unquoted string.
message: it's a value with an apostrophe
