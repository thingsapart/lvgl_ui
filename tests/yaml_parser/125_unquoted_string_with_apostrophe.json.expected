[{
		"message":	"it's a value with an apostrophe"
	}]
